core/eolearn/core/types.py

```diff
@@ -28,18 +28,10 @@
 
 # DEVELOPER NOTE: the #: comments are applied as docstrings
 
-#: Specification describing a single feature
-FeatureSpec: TypeAlias = Union[Tuple[Literal[FeatureType.BBOX, FeatureType.TIMESTAMPS], None], Tuple[FeatureType, str]]
-#: Specification describing a feature with its current and desired new name
-FeatureRenameSpec: TypeAlias = Union[
-    Tuple[Literal[FeatureType.BBOX, FeatureType.TIMESTAMPS], None, None], Tuple[FeatureType, str, str]
-]
-SingleFeatureSpec: TypeAlias = Union[FeatureSpec, FeatureRenameSpec]
+SingleFeatureSpec: TypeAlias = Union[Tuple[FeatureType, str], Tuple[FeatureType, str, str]]
```

Comment:
So, if I understand correctly, you decided to drop the simple `FeatureSpec` and `FeatureRenameSpec` because they are straightforward and don't need type aliases?

Reply:
yeah, we might consider having a `Feature` alias as we do in `eo-grow` just because it makes the signatures a bit nicer rather than `tuple[FeatureType, str]`

but the rename stuff can be dropped, almost never used anyway